### ISSUE TITLE
Fatal error initializing SatStack

### ISSUE THREAD

**Issue by serdiogo** [LedgerHQ/satstack#103]
FATAL Failed to initialize Bus                      error=bitcoind unreachable: invalid http POST response (nil), method: getblockchaininfo, id: 1, last error=Post "http://serdiogo.local/wallet/satstack": dial tcp: lookup serdiogo.local on [fe80::1%en14]:53: no such host

Hello, after warming up goes green, I get this message ...
anyone can help please ?

thanks in advance !

**Comment by adrienlacombe-ledger** [comment]
what kind of node are you running?

**Comment by serdiogo** [comment]
I'm on an iMac M1, with a VM (ubuntu) running umbrel.

**Comment by adrienlacombe-ledger** [comment]
Thank you, Umbrel usually uses Tor so you have to do this https://github.com/LedgerHQ/satstack#manual-setup-for-advanced-users replace serdiogo.local by the .onion address

**Comment by adrienlacombe-ledger** [comment]
Any issue, let me know, I will help :)

**Comment by serdiogo** [comment]
thanks again for your availability, that's awesome!

now I choose the rpc TOR from the node and put it on ledger live, but ledger now cannot sync with the node itself... is it another way around? :/

**Comment by adrienlacombe-ledger** [comment]
No, but this is fine, you won't be able to use Ledger Live to generate the config json file but you can do it manually as explained on GitHub and still make it work!

**Comment by serdiogo** [comment]
oh ok, I'll try for sure!

let me see and I come back to you!

**Comment by serdiogo** [comment]
iMac-de-Diogo ~ % cd scripts
cd: no such file or directory: scripts

when I try retrieve the scripts from the device my terminal gives this message ...

**Comment by serdiogo** [comment]
I already have the scripts external and internal but now can you simplify this step please ?

"Create a config file lss.json in your home directory. You can use [this](https://github.com/ledgerhq/satstack/blob/master/lss.mainnet.json) sample config file as a template.

Add "torproxy": "socks5://127.0.0.1:9050", to connect to a Tor client running locally so that satstack can reach a full node behind Tor. Replace the rpcurl with the .onion address of your node."

thanks a lot again.

**Comment by serdiogo** [comment]
I wrote the json file already, but now starting the satstack gives another error: 

FATAL Failed to load config                         error=config file not found

what should I do next? thanks !

**Comment by adrienlacombe-ledger** [comment]
@serdiogo your json config file must be located in the same folder as satstack executble

**Comment by adrienlacombe-ledger** [comment]
it must be named lss.json

**Comment by serdiogo** [comment]
I did that but same problem...

FATAL Failed to load config error=config file not found

**Comment by adrienlacombe-ledger** [comment]
make sure the json is properly formatted, no missing { or }, things like that...

**Comment by serdiogo** [comment]
<img width="569" alt="Captura de ecrã 2023-03-02, às 16 02 56" src="https://user-images.githubusercontent.com/125337077/222484602-20c140bc-2a32-4da0-96ed-6cd8f848b9a3.png">

I wrote it again, now on visual studio code, saved as lss.json on satstack folder and nothing yet.

**Comment by adrienlacombe-ledger** [comment]
let's take this via email if this is fine with you and we will come back here if we have valuable information to share with the world but I suspect this is just an issue with your local setup, you can reach me at adrien.lacombe at ledger.com
